confluent_kafka/src/Producer.c

```diff
@@ -260,25 +302,32 @@ static PyObject *Producer_produce (Handle *self, PyObject *args,
 			       "callback",
 			       "on_delivery", /* Alias */
 			       "partitioner",
+#if HAVE_PRODUCEV
+                               "timestamp",
```

Comment:
Is there any concern that users might write code but then if they run it against a build of the correct version of this library but an old version of librdkafka, they'll get weird errors about unexpected arguments? (I am, in particular, concerned that this could become a FAQ.) Also, we need to generate a single set of API docs, so users may become confused when they can't use a parameter that is listed in the docs.

Another option would be to always parse the argument, but throw an exception with a useful message (including listing what version of librdkafka they have).